pkgs/development/tools/database/sqlitebrowser/default.nix

```
@@ -1,30 +1,35 @@
-{ lib, stdenv, mkDerivation, fetchFromGitHub, cmake
-, qtbase, qttools, sqlcipher, wrapGAppsHook3, qtmacextras
+{ lib, stdenv, fetchFromGitHub, cmake
+, qtbase, qttools, sqlcipher, wrapQtAppsHook, qtmacextras
 }:
 
-mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "sqlitebrowser";
   version = "3.13.0";
 
   src = fetchFromGitHub {
-    owner = pname;
-    repo = pname;
-    rev = "v${version}";
+    owner = finalAttrs.pname;
```

```suggestion
    owner = "sqlitebrowser";
```

https://github.com/NixOS/nixpkgs/issues/277994

---

Thank you, done.